README.md

```diff
@@ -66,10 +66,10 @@ An instance of the Appcues SDK should be initialized when your app launches.
 ```js
 import * as Appcues from '@appcues/react-native'
 
-Appcues.setup('APPCUES_ACCOUNT_ID', 'APPCUES_APPLICATION_ID')
+await Appcues.setup('APPCUES_ACCOUNT_ID', 'APPCUES_APPLICATION_ID')
 ```
 
-Initializing the SDK requires you to provide two values, an Appcues account ID, and an Appcues mobile application ID. These values can be obtained from your [Appcues settings](https://studio.appcues.com/settings/account). Refer to the help documentation on [Registering your mobile app in Studio](https://docs.appcues.com/article/848-registering-your-mobile-app-in-studio) for more information.
+Initializing the SDK requires you to provide two values, an Appcues account ID, and an Appcues mobile application ID. These values can be obtained from your [Appcues settings](https://studio.appcues.com/settings/account). Refer to the help documentation on [Registering your mobile app in Studio](https://docs.appcues.com/article/848-registering-your-mobile-app-in-studio) for more information. Since the React Native bridge is asynchronous, the `setup` call should be awaited, before calling subsequent SDK functions.
```

Comment:
Yeah, I've never seen anywhere saying "awaited" indeed. I'd use something like:

> Since the React Native bridge is asynchronous, the `setup` call should be resolved before calling subsequent SDK functions.